examples/vm-two-nics/main.tf

```diff
@@ -0,0 +1,151 @@
+    description                = "Allow RDP access"
+    priority                   = 110
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = "*"
+    destination_port_range     = "3389"
+    source_address_prefix      = "*"
+    destination_address_prefix = "*"
+  }
+}
+
+# Create the first network interface card for External and attach the PIP and the NSG - this nic will have the deafult route
+resource "azurerm_network_interface" "ext-nic" {
+  name                      = "${var.prefix}-ext-nic"
+  location                  = "${azurerm_resource_group.main.location}"
+  resource_group_name       = "${azurerm_resource_group.main.name}"
+  network_security_group_id = "${azurerm_network_security_group.main.id}"
+
+  ip_configuration {
+    name                          = "primary"
+    subnet_id                     = "${azurerm_subnet.External.id}"
+    private_ip_address_allocation = "dynamic"
+    public_ip_address_id          = "${azurerm_public_ip.main.id}"
+  }
+}
+
+
+# Create the second network interface card for internal - no PIP and no NSG on this one
+resource "azurerm_network_interface" "int-nic" {
+  name                      = "${var.prefix}-int-nic"
+  location                  = "${azurerm_resource_group.main.location}"
+  resource_group_name       = "${azurerm_resource_group.main.name}"
+
+  ip_configuration {
+    name                          = "primary"
+    subnet_id                     = "${azurerm_subnet.Internal.id}"
+    private_ip_address_allocation = "dynamic"
+  }
+}
+
+resource "azurerm_virtual_machine" "main" {
+  name                  = "${var.prefix}-vm"
+  location              = "${azurerm_resource_group.main.location}"
+  resource_group_name   = "${azurerm_resource_group.main.name}"
+  primary_network_interface_id = "${azurerm_network_interface.ext-nic.id}"
+  network_interface_ids = ["${azurerm_network_interface.ext-nic.id}","${azurerm_network_interface.int-nic.id}"]
+  vm_size               = "Standard_DS1_v2"
+
+  # Uncomment this line to delete the OS disk automatically when deleting the VM
+  # delete_os_disk_on_termination = true
+
+  # Uncomment this line to delete the data disks automatically when deleting the VM
+  # delete_data_disks_on_termination = true
+
+  storage_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "16.04-LTS"
+    version   = "latest"
+  }
+
+  storage_os_disk {
+    name              = "${local.virtual_machine_name}-osdisk"
+    caching           = "ReadWrite"
+    create_option     = "FromImage"
+    managed_disk_type = "Standard_LRS"
+  }
+
+  os_profile {
+    computer_name  = "${local.virtual_machine_name}"
+    admin_username = "${var.admin_username}"
+    admin_password = "${var.admin_password}"
+  }
+
+  os_profile_linux_config {
+    disable_password_authentication = false
+  }
+}
+
+ 
```

Comment:
Please delete the last line with white space.

Reply:
done